modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/transformer/JournalTransformer.java

```diff
@@ -491,10 +491,14 @@ else if (type.equals(DDMFormFieldTypeConstants.SELECT) &&
 			while (iterator.hasNext()) {
 				Element optionElement = iterator.next();
 
-				dataJSONArray.put(optionElement.getData());
+				if (Validator.isNotNull(optionElement.getData())) {
```

Comment:
@DiegoHu97, no need to do this check. If there are no options this will never trigger. If there are options with empty values we should honor them